task/buildah-remote-oci-ta/0.4/buildah-remote-oci-ta.yaml

```diff
@@ -400,6 +400,16 @@ spec:
       chown root:root /var/lib/containers
 
       sed -i 's/^\s*short-name-mode\s*=\s*.*/short-name-mode = "disabled"/' /etc/containers/registries.conf
+      cat << 'EOF' >> /etc/containers/registries.conf
+      [[registry]]
+      prefix   = "registry.redhat.io"
```

Comment:
Why do we need this? It's recommended to use FQDN in Containerfiles.

Also the idea of hardcoding RH specific paywall registry in upstream project, isn't something, I'm in favor of.